libs/state/src/lib/rx-state.service.ts

```diff
@@ -78,6 +79,10 @@ export class RxState<T extends object> implements OnDestroy, Subscribable<T> {
     this.subscription.unsubscribe();
   }
 
+  setAccumulator(accumulatorFn: AccumulationFn) {
```

Comment:
we need to test this method, otherwise it's fine i guess :)

Reply:
added tests